owltorrent-common/src/main/java/edu/rice/owltorrent/common/entity/Bitfield.java

```diff
@@ -0,0 +1,56 @@
+package edu.rice.owltorrent.common.entity;
+
+import java.util.BitSet;
+
+public class Bitfield {
+  public BitSet bitSet;
+  /** For peers who don't send a bitfield message. */
+  public static final EmptyBitfield EMPTY_BITFIELD_SINGLETON = new EmptyBitfield();
```

Comment:
Nit: The type should be Bitfield instead, since the EmptyBitfield class is private?